static/docs/user-guide/external-data/local.md

```diff
@@ -0,0 +1,311 @@
+# External Data on a Local Filesystem
+
+Sometimes we would like to keep the data and cache outside the project
+directory, so that we can search easily the code of the project.
+
+Keeping the data outside of the project, on a fixed absolute path, is also
+useful in case that there are several projects that use the same data.
+
+Another case when this would be desirable is if we want to keep the data on an
+[external drive](https://whatis.techtarget.com/definition/external-hard-drive)
+because the data is too big to fit on our home directory, or for some other
+reasons.
+
+A similar case is also when the data is on a
+[network-attached storage (NAS)](https://searchstorage.techtarget.com/definition/network-attached-storage)
+and is mounted through NFS, SSHFS, Samba, etc.
+
+## Tracking External Data
+
+Let's say that we want to track the directory `/var/data/mydata/`. We can simply
+do it like this:
+
+```dvc
+$ dvc add /var/data/mydata/
+```
+
+This will create the tracking file `mydata.dvc` with a content like this:
+
+```yaml
+outs:
+  - path: /var/data/mydata/
+    metric: false
+    cache: true
+    persist: false
+    md5: 800be79fa2d0c411def72cd04e8600c8.dir
+md5: 22ee63d16fe8c622fd1a4f138533b036
+```
+
+<details>
+
+### Using an environment variable
```

Comment:
these hack apply to any storage type, I would generalize and move them somewhere ... not sure actually where and how yet

Reply:
> these hack apply to any storage type, I would generalize and move them somewhere ... not sure actually where and how yet

You are right, it applies to the other storage types too, but I don't see a better place either. The "index" page would not be a good place because you need to explain first the normal way, before explaining the hack (or the tip, or the optimization). Let's leave it on the local page and the user can generalize it for the other storage types himself.

By the way, the "local" storage type applies to all the users, so all of them need to read it. The other storage types don't apply (or are not relevant) to all the users. For example a user may use S3, but not GS etc.

Reply:
My point is that these pages should be very simple. Most of the information should be generalized and provided a two pages - external dependencies and external outputs.

> Let's leave it on the local page and the user can generalize it for the other storage types himself.

I don't see this as a reasonable solution.

> By the way, the "local" storage type applies to all the users

May be I'm not following something, but no, external "local" data management is a very specific case. Mostly happening due to misunderstanding of the DVC default setup - shared cache.

Reply:
"shared cache" itself is a case of "local" external data (since the cache is outside the project directory).